Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -375,7 +375,7 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"For further information, please see our FAQ page."</string>
     <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/en/faq/#encounter_but_green">For further information, please see our FAQ page.</a>"</string>
+    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">"For further information, please see our FAQ page."</a></string>
```

Comment:
@janetback I copied this link from the EN translation. It seems odd though, that the language tag in the Link is DE instead of EN.